lesson_10/libraries/src/loaders/loaders.module.ts

```diff
@@ -1,11 +1,16 @@
 import { Module } from '@nestjs/common';
 import { AnthonyMaysLoader } from './anthony_mays_loader.js';
+import { LjMcwilliamsLoader } from './lj_mcwilliams_loader.js';
 import { XavierCruzLoader } from './xavier_cruz_loader.js';
 
 export const Loaders = Symbol.for('Loaders');
 
 // Add your quiz provider here.
-const LOADER_PROVIDERS = [AnthonyMaysLoader, XavierCruzLoader];
+const LOADER_PROVIDERS = [
```

Comment:
Need to update following recent submits.

Reply:
Just resolved merge conflicts